protocol/messenger_communities.go

```diff
@@ -792,9 +792,9 @@ func (m *Messenger) requestCommunityInfoFromMailserver(communityID string) (*com
 		nil,
 		nil,
 		filter,
-		true)
+		waitForResponse)
 
-	if err != nil {
+	if err != nil || !waitForResponse {
```

Comment:
here you want to always return the error, sorry for the confusion if I asked you to change it before, I must have misread it.

Essentially:


``` 
if err != nil {
  return nil, err
}

if !waitForResponse {
return nil, nil
}

```

Reply:
or I guess it's the same as you wrote, I need to go to bed :) nevermind

Reply:
Alright, it's back to the way it was before, the way you commented